src/components/Calculator.js

```diff
@@ -1,39 +1,58 @@
 // Calculator.js
```

Comment:
- [ ] It would be nice if you don't leave the commented piece of code in your project to follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 😊